precompiles/xcm/src/tests.rs

```diff
@@ -108,7 +110,7 @@ fn remote_transact_works() {
 }
 
 #[test]
-fn reserve_transfer_works() {
+fn reserve_transfer_non_native_works() {
```

Comment:
Perhaps a bit misleading - it's still native but not native currency. All of these assets are in fact _native_ and our parachain acts as reserve for them.

Reply:
Feel free to suggest better name. Naming is hard.